src/org/ice1000/tt/editing/agda/lex-highlight.kt

```diff
@@ -51,7 +51,10 @@ object AgdaHighlighter : AgdaGeneratedSyntaxHighlighter() {
 		AgdaTypes.KW_DATA,
 		AgdaTypes.KW_LET,
 		AgdaTypes.KW_IN,
-		AgdaTypes.KW_DO
+		AgdaTypes.KW_DO,
+		AgdaTypes.KW_HIDING,
+		AgdaTypes.KW_RENAMING,
+		AgdaTypes.KW_USING
```

Comment:
Next time you can add the three new items before `DO` to avoid one line of git change (I'm not asking you to change anything, just FYI)